### DISCUSSION TITLE
How to enable serde as a user?

### DISCUSSION THREAD

**Discussion by Samrose-Ahmed** [smithy-lang/smithy-rs#2927] [Q&A]
I am currently using smithy-rs to generate a rust server and client from a Smithy model. I believe support was recently added for serde behind a flag (?), how would I enable this? I am currently using a submodule and the build.gradle.kts with the smithy-build.json.

**Comment by jdisanti** [comment]
Currently the support is only for the types in the `aws-smithy-types` crate. You can enable it via a Rust compiler flag: `--cfg aws_sdk_unstable` _and_ by enabling the `serde-serialize` and `serde-deserialize` features. The generated code itself doesn't have any serde support yet.
